application/core/database/cycle-stat.go

```diff
@@ -0,0 +1,40 @@
+package database
+
+import (
+	"github.com/google/uuid"
+	"github.com/koki/randommatch/entity"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+func CreateStat(cycle entity.MatchingCycle) (string, error) {
+	driver, err := Driver()
+	if err != nil {
+		return "", err
+	}
+	session := (*driver).NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	uid, err := session.WriteTransaction(func(tx neo4j.Transaction) (interface{}, error) {
+		result, err := tx.Run("MERGE (n:Stat {uuid: $id}) "+
+			"ON CREATE SET n += {uid: $id, "+
```

Comment:
you do not need to have uid as part of the set since it is already put on the previous line